src/codegeneration/ClassTransformer.js

```diff
@@ -337,10 +371,15 @@ export class ClassTransformer extends TempVarTransformer{
     return createParenExpression(this.makeStrict_(expression));
   }
 
-  transformPropertyMethodAssignment_(tree, homeObject, internalName) {
+  transformPropertyMethodAssignment_(tree, homeObject, internalName, originalName) {
     var parameterList = this.transformAny(tree.parameterList);
     var body = this.transformSuperInFunctionBody_(
         tree.body, homeObject, internalName);
+
+    if (this.showDebugNames_) {
+      tree.debugName = classMethodDebugName(originalName, methodNameFromToken(tree.name), isStatic);
```

Comment:
I'm getting `Error: Not yet implemented
    at LiteralPropertyName.getStringValue (/Users/vojta/Code/traceur/bin/traceur.js:5481:13)`.